web/app/room/[roomId]/page.jsx

```
@@ -369,26 +649,314 @@ export default function RoomPage({ params }) {
+      {verifying && (
+        <div className="fixed inset-0 bg-black/50 flex items-center justify-center p-4 z-50">
+          <Card className="w-full max-w-md">
+            <CardContent className="p-6 space-y-4">
+              <h3 className="text-lg font-semibold">Verify Submission</h3>
+              <p className="text-xs font-mono text-muted-foreground break-all">
+                {verifying.submission_id}
+              </p>
+              <form onSubmit={onVerifySubmit} className="space-y-4">
+                <div>
+                  <label className="text-sm font-medium">Claim (optional)</label>
+                  <select name="claim_id" className="w-full mt-1 border rounded p-2 bg-background">
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)